### DISCUSSION TITLE
There should probably be a big download button somewhere on the mod page

### DISCUSSION THREAD

**Discussion by coderbot16** [modrinth/.github#53] [Feature requests]
When users navigate to a mod page, it could be difficult for them to figure out where the button to actually download the mod is. I think that it would be good to have a big download button that would immediately download whatever the "best" version of the mod is (not sure how to figure that out).

For example, a big download button could be placed on the right of this part of the mod page:

![image](https://user-images.githubusercontent.com/22420959/114278004-8f411d80-99e2-11eb-9b21-5db587562c38.png)

I feel like this would make Modrinth much more intuitive to users, who expect to just have a big download button that they can click instead of having to figure out that the Versions section is where the downloads are.

**Comment by triphora** [comment]
This seems like it could go wrong in too many ways. It encourages users to disregard anything relating to loaders, Minecraft version, etc. The featured versions section also does this rather well, I think, as it has the download links for major versions and for each of the loaders right there on the side. I'm going to keep this open to get feedback from others, though.

**Comment by coderbot16** [comment]
Maybe it would be possible to have such a button provide selections for mod loader / mc version / etc when clicked - so not an immediate download button, and not anything you can't get in the versions section.

Basically, that idea would be something that immediately jumps out as "I want the latest version for my install" and then clearly presents a dialog for getting the needed information out of the user to disambiguate what "latest" version to present. Might end up being an improvement for newer users who might otherwise get lost searching for a clear "Download" button.

**Comment by Secret-chest** [comment]
Maybe something like this?:
* there is a big download button
* you have a dropdown for  downloading for other MC versions
* if the mod has both fabric/forge, there is a green button for fabric and a blue one for forge
* below the button(s), it says something like this: "For Fabric 1.18.2. Requires Fabric API."

**Comment by carterprince** [comment]
Maybe there could be a button that could open a Modrinth-protocol link, for example something like `mod://AANobbMI/versions/mc1.18.2-0.4.1/sodium-fabric-mc1.18.2-0.4.1%2Bbuild.15.jar`, which could then be handled by an external program, allowing for one-click installations from Modrinth. Just a thought.

**Comment by triphora** [comment]
@SoopaKhell This is actually already being implemented in #417!
